mcodingbot/tasks/stats.py

```diff
@@ -122,27 +121,32 @@ async def get_stats(bot: Bot) -> Stats:
     return _last_known_stats
 
 
+def format_float(n: int | float) -> int | float:
```

Comment:
Would rename this to `strip_trailing_zeros` tbh

Reply:
Yeah sounds more accurate, thanks!